charts/datadog/templates/_helpers.tpl

```diff
@@ -893,3 +893,16 @@ Create RBACs for custom resources
     false
   {{- end -}}
 {{- end -}}
+
+{{/*
+Return all namespaces with disabled Single Step Instrumentation
+*/}}
+{{- define "apmInstrumentation.disabledNamespaces" -}}
+{{- if and .Values.datadog.apm .Values.datadog.apm.instrumentation -}}
+{{- if and .Values.datadog.apm.instrumentation.disabledNamespaces .Values.datadog.apm.instrumentation.enabled -}}
```

Comment:
`if and .Values.datadog.apm.instrumentation.disabledNamespaces`: does this mean that a user has to define some disabled namespaces in order for us to append the Agent namespace? If the user doesn't specify any disabled namespaces we should still ignore the Agent namespace, right?

I'm not sure if ` .Values.datadog.apm.instrumentation.disabledNamespaces` defaults to a truthy value here if it is not set 🤔

Reply:
So the logic is folowing:
```
if disabledNamespace is set AND apm instrumentation enabled
  append datadog namespace to disabledNamespace and return as a new list
else if apm instrumentation enabled
  return datadog namespace
end
```

Reply:
Oh I see, because of the `else if` and line 905 we default with the datadog namespace!

Do we need to normalize the agent namespace to a list or does the code handle a single value?